android/app/src/main/jni/Application.mk

```diff
@@ -0,0 +1,2 @@
+APP_ABI := armeabi
```

Comment:
Also add armeabi-v7a and x86 here :)

And inside the Android.mk you would then use something like this, and set those three variables:

ifeq ($(TARGET_ARCH_ABI),armeabi)
GSTREAMER_ROOT        := $(GSTREAMER_ROOT_ARM)
else ifeq ($(TARGET_ARCH_ABI),armeabi-v7a)
GSTREAMER_ROOT        := $(GSTREAMER_ROOT_ARMV7)
else ifeq ($(TARGET_ARCH_ABI),x86)
GSTREAMER_ROOT        := $(GSTREAMER_ROOT_X86)
else
$(error Target arch ABI not supported)
endif